lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -126,9 +135,12 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                         .Set(AzureBatchRuntimeClientConfiguration.AzureStorageContainerName, @"###########")
                         //// Extend default retry interval in Azure Batch
                         .Set(AzureBatchRuntimeClientConfiguration.DriverHTTPConnectionRetryInterval, "20000")
-                        //// To allow Driver - Client communication, please specify the ports to use to set up driver http server.
-                        //// These ports must be defined in Azure Batch InBoundNATPool.
-                        .Set(AzureBatchRuntimeClientConfiguration.AzureBatchPoolDriverPortsList, new List<string>(new string[] { "123", "456" }))
+                        .Set(AzureBatchRuntimeClientConfiguration.AzureBatchPoolDriverPortsList, ports)
+                        // Bind to Container Registry properties if present
```

Comment:
I think it might be a good idea to comment this line by default as what you do for container.

Reply:
You mean make it //// instead just of // for this line?

---
In reply to: [202416284](https://github.com/sharathmalladi/reef/pull/5#discussion_r202416284) [](ancestors = 202416284)